main.cc

```diff
@@ -293,6 +296,10 @@ init()
 
     // Absorb all the init time so we dont try to catch up
     frame_info.tick();
+
+    audio = new SoLoud::Soloud();
```

Comment:
init before the tick() call just above -- otherwise sim has to catch up all the time the init takes.